### PR TITLE
🌐📝 - ADD Network policy chart

### PR DESCRIPTION
**Pull request by springdo**
FYI - work from today's sesh ....   
@rh-jpoole @ckavili

### CODE REVIEW

File: charts/network-policy/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# 🌐📝 Network Policy
+
+Network policy chart for creating rules on Network traffic in a `Namespace`. This chart's defaults contains an example for multientant ingress for a namespace by denying all traffic, then allowing specific pod to pod communication and ingress traffic in from the openshift routers' namespace. An example `policy` is also included to allow prometheus scrape metrics from the project.
```
Comment (garethahealy) [inline charts/network-policy/README.md:3]: multientant?

Comment (springdo) [reply charts/network-policy/README.md:3]: yeah
it's the future of `multitenant`

File: charts/network-policy/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+The above command creates objects with default configuration.
+
+To run the chart and remove some of the configuration, for example the `multitenant_policies`:
+```bash
+$  helm template my-policy -f network-policy/values.yaml network-policy --set multitenant_policies= | oc apply -n my-project -f-
+```
+
+
+## Configuration
+The following table lists the configurable parameters of the Bootstrap chart and their default values.
```
Comment (garethahealy) [inline charts/network-policy/README.md:22]: Bootstrap chart: copy and paste?

**Review by garethahealy — APPROVED** [review]

**Comment by garethahealy** [comment]
lgtm - will let @ckavili merge since you've called her out in the description.

**Comment by ckavili** [comment]
> lgtm - will let @ckavili merge since you've called her out in the description.

thanks :)

**Pull request merged**